common/test/TestLogging.cs

```diff
@@ -93,30 +96,34 @@ public void WriteLine(string message, SeverityLevel severity = SeverityLevel.Inf
                     propertyBag = new Dictionary<string, string>();
```

Comment:
Use this other ctor. https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.-ctor?view=netcore-3.1#System_Collections_Generic_Dictionary_2__ctor_System_Collections_Generic_IEnumerable_System_Collections_Generic_KeyValuePair__0__1___

Reply:
Actually, you are still doing this wrong.

Don't allocate a new dictionary and copy items if there are not trace-specific custom properties. Just use the CommonProperties dictionary.

If they do pass a dictionary, then create a new one and use the ctor I linked so you do not modify the dictionary they sent. Then add the common properties. Or do it the other way around.